web/beacon-app/src/features/topics/components/Modal/RevokeAPIKeyModal.tsx

```diff
@@ -92,15 +92,13 @@ const RevokeAPIKeyModal = ({ onOpen, onClose }: RevokeAPIKeyModalProps) => {
         <div className="pb-6" data-cy="api-key-name">
           <span className="font-bold">Key Name:</span> {key?.name}
         </div>
-        <CheckboxFieldset onClick={handleCheckboxChange} className="pb-8" data-cy="revoke-checkbox">
-          <div className="flex items-start">
-            <Checkbox></Checkbox>
-            <Trans>
-              I understand that revoking the API key will cause publishers and subscribers to lose
-              access to the event stream (topic) and may impact performance.
-            </Trans>
-          </div>
-        </CheckboxFieldset>
+        <Checkbox
```

Comment:
this is much more easier to follow with each property separated into its own line